newIDE/app/src/EventsSheet/EventsTree/DropContainer.js

```diff
@@ -0,0 +1,295 @@
+// @flow
+import * as React from 'react';
+import { getIndentWidth, type SortableTreeNode } from '.';
+import {
+  moveNodeAbove,
+  moveNodeBelow,
+  moveNodeAsSubevent,
+  isSameDepthAndBelow,
+} from './helpers';
+import { type WidthType } from '../../UI/Reponsive/ResponsiveWindowMeasurer';
+import './style.css';
+import GDevelopThemeContext from '../../UI/Theme/ThemeContext';
+
+const styles = {
+  dropArea: { zIndex: 1, position: 'absolute' },
+  dropIndicator: {
+    position: 'absolute',
+    zIndex: 2,
+    border: '4px solid black',
+    outline: '1px solid white',
+  },
+};
+
+const getTargetPositionStyles = (
+  indentWidth: number,
+  draggedNodeHeight: number,
+  isDraggedNodeChild: boolean
+) => ({
```

Comment:
Is it worth mentionning it returns `{[position: string]; DropTargetContainerStyle}` ?